.github/workflows/build-and-test.yml

```diff
@@ -1,6 +1,9 @@
 ﻿name: build and test
-on: [push, pull_request]
-
+on:
+  pull_request:
+  push:
+    tags-ignore:
```

Comment:
shouldn't this be `tags` and not `tags-ignore` as to only trigger on pr w/ tag "`Backend/*`"